packages/fxa-shared/db/models/auth/email-bounce.ts

```diff
@@ -37,20 +38,28 @@ export class EmailBounce extends BaseAuthModel {
 
   // table fields
   email!: string;
+  emailTypeId!: number;
   bounceType!: number;
   bounceSubType!: number;
   createdAt!: number;
 
   static async create({
     email,
+    templateName,
     bounceType,
     bounceSubType,
   }: Pick<EmailBounce, 'email'> & {
+    templateName: string;
     bounceType: BounceType;
     bounceSubType: BounceSubType;
   }) {
+    const emailTypes = EmailType.bindKnex(this.knex);
```

Comment:
Feeling a little uncertain about this guy here 🤔

Reply:
Me to 😅. Could you use the stored procedure to create the bounce instead? I'm not sure what the standard is since I noticed it is done both ways in other areas.